exercises/practice/custom-set/.meta/src/CustomSet.example.elm

```diff
@@ -0,0 +1,198 @@
+module CustomSet exposing (diff, disjoint, empty, equal, fromList, insert, intersect, isEmpty, member, subset, toList, union)
```

Comment:
This seems like a complicated example! It seems like there would be a simpler solution. Not sure if that matters or not.

Reply:
Of course, the instructions say that anything is fine, so it would be easier to model a set as a list, but where's the fun in that? 😆 
I was actually thinking that my solution was too simple because my tree is unbalanced, but implementing a red-black tree might be a bit much.

Ultimately, like you are saying, it doesn't matter, this is not a reference solution, just a proof that it can be solved in Elm.